### PR TITLE
chore: use spec-prod to publish

### LINKED ISSUE #144: TravisCI build error: The command "bikeshed update" failed and exited with 1 during .
https://github.com/w3c/clipboard-apis/pull/143 has a build error which looks like an infra problem:
https://travis-ci.com/github/w3c/clipboard-apis/builds/226470464

$ bikeshed update
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/pkg_resources/__init__.py", line 568, in _build_master
    ws.require(__requires__)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/pkg_resources/__init__.py", line 886, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/pkg_resources/__init__.py", line 777, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (typing-extensions 3.10.0.0 (/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages), Requirement.parse('typing-extensions<3.8,>=3.7.4.1'), {'widlparser'})

The command "bikeshed update" failed and exited with 1 during .

### PR DESCRIPTION
**Pull request by marcoscaceres**
cc @sidvishnoi for a look over. 

Closes #144

### CODE REVIEW

File: .github/workflows/pr-push.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# .github/workflows/pr.yml
```
Comment (sidvishnoi) [inline .github/workflows/pr-push.yml:1]:
```suggestion
```

File: .github/workflows/pr-push.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+name: CI
+on:
+  pull_request: {}
+  push:
+    branches: [main]
+jobs:
+  main:
+    name: Build, Validate, Deploy
+    runs-on: ubuntu
```
Comment (sidvishnoi) [inline .github/workflows/pr-push.yml:10]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/workflows/pr-push.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  main:
+    name: Build, Validate, Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: w3c/spec-prod@v2
+       with:
+          GH_PAGES_BRANCH: gh-pages
+          W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
+          W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-webapps/2014JulSep/0627.html
```
Comment (sidvishnoi) [inline .github/workflows/pr-push.yml:16]: Would need `W3C_BUILD_OVERRIDE` also:
```yml
W3C_BUILD_OVERRIDE: |
  status: WD
```

**Review by sidvishnoi — APPROVED** [review]
LGTM.

**Comment by marcoscaceres** [comment]
Alright, let's ship it... will then turn on checks.

**Comment by annevk** [comment]
\o/

**Comment by marcoscaceres** [comment]
@garykac, I've set it to complain if there are linking errors. There are some already, so any subsequent pull request/merges will complain.

**Pull request merged**